jwst/saturation/tests/test_saturation.py

```diff
@@ -42,6 +42,70 @@ def test_basic_saturation_flagging(setup_nrc_cube):
     assert np.all(output.groupdq[0, satindex:, 5, 5] == dqflags.group['SATURATED'])
 
 
+def test_low_saturation_flagging(setup_nrc_cube):
+    """Check that the saturation flag is set when a pixel value is zero or
+    negative."""
+
+    # Create inputs, data, and saturation maps
+    ngroups = 5
+    nrows = 2048
+    ncols = 2048
```

Comment:
Don't need 2k rows and columns when only pixel 5,5 is being checked in the test. Could make it much smaller like 10 or 20 to save time when running the test.

Reply:
Done.